modules/007-registrypackages/images/iptables/scripts/install

```diff
@@ -66,6 +69,22 @@ function checkNFTSupportByAddingNFTRules() {
   return 1
 }
 
+function check_python() {
```

Comment:
seems overengineered.
mb bash version?
https://stackoverflow.com/a/4025065

Reply:
This bash solution is bigger and less clear (IMHO). Python is a required dependency for DKP and I think it is better to use it.